go/client/commands_common.go

```diff
@@ -25,6 +25,7 @@ func GetCommands(cl *libcmdline.CommandLine, g *libkb.GlobalContext) []cli.Comma
 		NewCmdDb(cl, g),
 		NewCmdDevice(cl, g),
 		NewCmdID(cl, g),
+		NewCmdInstall(cl, g),
```

Comment:
Seems like we need another refactor here.  On Linux and Windows, we don't want this command showing up at all, and we want `keybase install` to give an error.  So we should make a `commands_osx.go` and augment the list there.